src/backend/marsha/core/serializers/video.py

```diff
@@ -486,17 +486,18 @@ def get_xmpp(self, obj):
         Dictionnary
             A dictionary containing all info needed to manage a connection to a xmpp server.
         """
-        if (
-            settings.LIVE_CHAT_ENABLED
-            and obj.live_state is not None
-            and self.context.get("user_id", False)
-        ):
+        user_id = (
+            self.context.get("user_id")
+            if self.context.get("user_id", None) is not None
+            else self.context.get("session_id", None)
+        )
```

Comment:
Maybe clearer like this?
```suggestion
        user_id = self.context.get("user_id") or self.context.get("session_id")
        )
```

Reply:
or maybe
```suggestion
        user_id = self.context.get("user_id", self.context.get("session_id"))
```

Reply:
@kernicPanel this version doesn't work, `user_id` can exists and be `None`

Reply:
oh, ok